### ISSUE TITLE
Fails to build since 1.43.0

### ISSUE THREAD

**Issue by Gerschtli** [pietdevries94/playwright-web-flake#3]
Hey, I noticed that playwright-test fails to build since 1.43.0.

```sh
$ nix shell github:pietdevries94/playwright-web-flake/1.43.0#playwright-test -L
playwright-driver> unpacking sources
playwright-driver> unpacking source archive /nix/store/fvlmg71izw7x4pfriqkizb5h0xxg5zgd-playwright-1.43.0-linux.zip
playwright-driver> source root is .
playwright-driver> setting SOURCE_DATE_EPOCH to timestamp 1712265243 of file ./package/types/types.d.ts
playwright-driver> patching sources
playwright-driver> substitute(): ERROR: file 'playwright.sh' does not exist
playwright-driver> /nix/store/7aprjn0yf6p06krg4087sxvilfp4c8yy-stdenv-linux/setup: line 144: pop_var_context: head of shell_variables not a function context
error: builder for '/nix/store/h3yljapy5yl2gj95l62z8rknfbhg1qqm-playwright-driver-1.43.0.drv' failed with exit code 1;
       last 7 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/fvlmg71izw7x4pfriqkizb5h0xxg5zgd-playwright-1.43.0-linux.zip
       > source root is .
       > setting SOURCE_DATE_EPOCH to timestamp 1712265243 of file ./package/types/types.d.ts
       > patching sources
       > substitute(): ERROR: file 'playwright.sh' does not exist
       > /nix/store/7aprjn0yf6p06krg4087sxvilfp4c8yy-stdenv-linux/setup: line 144: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/h3yljapy5yl2gj95l62z8rknfbhg1qqm-playwright-driver-1.43.0.drv'.
error: 1 dependencies of derivation '/nix/store/gwwjv1axcm067ik6gp17ivk2pq0svgph-playwright-browsers-chromium.drv' failed to build
error: 1 dependencies of derivation '/nix/store/1ha1h653pmfb6hv1qqd0k4031nsfzj6n-_at_playwright_slash_test-1.43.0.drv' failed to build
```

**Comment by thenbe** [comment]
Yeah I remember encountering this when I tried to upgrade. Error originates here: https://github.com/pietdevries94/playwright-web-flake/blob/a261a2932ad50baa9f7f45b3fe1865265d1701d3/playwright-driver/default.nix#L49-L59

**Comment by pietdevries94** [comment]
Thanks for reporting. The project in which we use this flake ourselves is still on 1.42 so we didn't encounter this issue.

I have "fixed" the problem in a PR but I can't fully test it. If either of you use macOS, I would love your help. Otherwise I will merge the fix monday. If you can't wait, the branch in the PR works on Linux, afaik.

https://github.com/pietdevries94/playwright-web-flake/pull/4

**Comment by pietdevries94** [comment]
@Gerschtli @thenbe Thanks to both of your help, the issue is fixed and there is a new tag for 1.43.1 which should work as intended.

I have removed the tag for 1.43.0, which has never worked and I assume nobody will require it. If someone does need it for whatever reason, I will back port it. :)
